sdk/core/azure-core-test/src/main/java/com/azure/core/test/utils/TestProxyManager.java

```diff
@@ -20,30 +20,27 @@
 /**
  * Manages running the test recording proxy server
  */
-public class TestProxyManager {
+public final class TestProxyManager {
     private static final ClientLogger LOGGER = new ClientLogger(TestProxyManager.class);
-    private Process proxy;
-    private final Path testClassPath;
+    private static Process proxy;
+    private static final Path WORKING_DIRECTORY = Paths.get(System.getProperty("user.dir"));
```

Comment:
I tested this running in IntelliJ as well as `mvn` from both the SDK directory and the root of the repo using `-pl`.

Reply:
I think it is consistent enough for our purposes - java.exe (through Maven typically) will be started in some path in the repo, which is all we need. My goal is to stop using the `target` dir here as it isn't something we need in this class so plumbing it in isn't necessary. Happy to hear of another alternative.